README.rst

```diff
@@ -523,8 +525,6 @@ Choose from the list of available configurations:
 The ``--dry-run`` option displays the files that need to be
 fixed but without actually modifying them:
 
-.. code-block:: bash
```

Comment:
@vinkla this configuration is related to your Git setup. PhpStorm does not install git by itself. It relies on the git executable available in your system.
You probably have made such a configuration during the installation of git (even if you don't remember it).

I suggest you to run ``git config --global core.autocrlf input`` to change the config so that it converts newlines to LF when committing (ensuring you don't submit windows newlines by mistakes) but keeps unix newlines on checkout, ensuring you don't alter the library code (this is the difference with configuring it to ``true``, which will convert both ways).

You can run ``git config --get core.autocrlf`` to see what is the current config value